src/translations/sk.json

```diff
@@ -0,0 +1,41 @@
+{
+  "log_in": "Prihlásiť sa",
+  "log_out": "Odhlásiť sa",
+  "logged_in_as": "Prihlásený ako",
+  "logged_in": "Prihlásený užívateľ",
+  "logging_in": "Prebieha prihlásenie",
+  "logging_out": "Prebieha odhlásenie",
+  "sign_up": "Zaregistrovať sa",
+  "signing_up": "Registrácia",
+  "forgot_password": "Zabudli ste heslo?",
+  "recover_password": "Obnoviť heslo",
+  "send_recovery_email": "Odoslať e-mail pre obnovenie",
+  "sending_recovery_email": "Odosielanie e-mailu pre obnovenie",
+  "never_mind": "Naspäť",
+  "update_password": "Aktualizovať heslo",
+  "updated_password": "Aktualizácia heslá",
+  "complete_your_signup": "Dokončite registráciu",
+  "site_url_title": "Nastavenie vývoja",
+  "site_url_link_text": "Vymazať URL localhost",
+  "site_url_message": "Zdá sa, že používate lokálny server. Povedzte nám adresu URL svojho Netlify servera.",
+  "site_url_label": "Zadajte URL svojho servera Netlify",
+  "site_url_placeholder": "URL vášho Netlify servera",
+  "site_url_submit": "Nastaviť adresu URL",
```

Comment:
```suggestion
  "site_url_submit": "Nastaviť URL adresu",
```

Reply:
Not sure why I cannot confirm this one 😔

![image](https://user-images.githubusercontent.com/1215304/120681908-2c2cae00-c49c-11eb-8eae-8277764e3cb9.png)

Reply:
It says it's because it's outdated 😓  Sorry about that, would it be because I commented on an older commit?

Reply:
Lemme re-suggest it

Reply:
Oh nevermind, looks like you fixed it! Thank you.